### ISSUE TITLE
Can't set quality level if PNG selected

### ISSUE THREAD

**Issue by MtnBiker** [nextgis/QTiles#79]
You can't set the quality level if PNG selected. However you can if JPG is selected. Current version of QTiles. QGIS 2.18.2 on macOS.

**Comment by SpiritQuaddicted** [comment]
PNG is a lossless compression algorithm while the quality setting applies to JPEG's lossy compression. Thus it makes no sense to have a quality level for PNG.

It would be nice to be able to set the compression rate (smaller size, slower creation) though, I will open an issue if there is not one already.

This issue though can be closed. :)

**Comment by kannes** [comment]
PNG is a lossless compression algorithm while the quality setting applies to JPEG's lossy compression. Thus it makes no sense to have a quality level for PNG.

It would be nice to be able to set the compression rate (smaller size, slower creation) though, I will open an issue if there is not one already.

This issue though can be closed. :)

**Comment by StacktraceException** [comment]
no, the current best image format is JPEG XL for a multitude of reasons. Among which is also the best compression ratio for both lossy and lossless images. png and jpeg are legacy outdated formats. they are unacceptable in 2023.

we should focus on current technology instead of decades old.

**Comment by MtnBiker** [comment]
How well supported is this and what's its future: "Besides [Cloudinary](https://en.wikipedia.org/wiki/Cloudinary) and [Google](https://en.wikipedia.org/wiki/Google) (originally), throughout JPEG XL's preliminary implementation in web browsers, various representatives of well-known industry brand names have publicly voiced support for JPEG XL as their preferred choice, including [Facebook](https://en.wikipedia.org/wiki/Facebook),[[24]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-25)[[25]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-26) [Adobe](https://en.wikipedia.org/wiki/Adobe_Inc.),[[26]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-27)[[27]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-28) [Intel](https://en.wikipedia.org/wiki/Intel) and the [Video Electronics Standards Association](https://en.wikipedia.org/wiki/Video_Electronics_Standards_Association),[[28]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-29)[[29]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-30) [The Guardian](https://en.wikipedia.org/wiki/The_Guardian),[[30]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-31)[[31]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-32) [Flickr](https://en.wikipedia.org/wiki/Flickr) and [SmugMug](https://en.wikipedia.org/wiki/SmugMug),[[32]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-33) [Shopify](https://en.wikipedia.org/wiki/Shopify),[[33]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-34) the [Krita](https://en.wikipedia.org/wiki/Krita) Foundation,[[34]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-35) and [Serif Ltd.](https://en.wikipedia.org/wiki/Serif_Europe)[[35]](https://en.wikipedia.org/wiki/JPEG_XL#cite_note-36)" --Wikipedia

**Comment by kannes** [comment]
Don't waste any time on feeding an obnoxious user who barged in here without any manners or respect. JPEG XL is amazing but this issue is not even remotely about it.
